src/website/views/components/toolbar/_examples.pug

```diff
@@ -1,499 +1,1371 @@
+      :code(lang='html')
+        <div class="chi-toolbar -portal">
+          <div class="chi-toolbar__header">
+            <div class="chi-toolbar__start">
+              <div class="chi-toolbar__search">
+                <div class="chi-form__item">
+                  <chi-search-input portal="portal" size="lg" id="example__base-3"></chi-search-input>
+                </div>
+              </div>
+              <div class="chi-divider -vertical"></div>
+              <div class="chi-toolbar__filters">
+                <div class="chi-toolbar__filters-desktop">
+                  <div class="chi-form__item">
+                    <select class="chi-select -lg -portal" id="example-ba5">
+                      <option value="">Select</option>
+                      <option>Option 1</option>
+                      <option>Option 2</option>
+                      <option>Option 3</option>
+                    </select>
                   </div>
-                </button>
-                <button class="chi-button -icon -flat" id="button-br-refresh" aria-label="Refresh" data-tooltip="Refresh" data-position="top">
-                  <div class="chi-button__content">
-                    <i class="chi-icon icon-refresh"></i>
+                  <div class="chi-form__item">
+                    <select class="chi-select -lg -portal" id="example-ba6">
+                      <option value="">Select</option>
+                      <option>Option 1</option>
+                      <option>Option 2</option>
+                      <option>Option 3</option>
+                    </select>
                   </div>
-                </button>
-                <button class="chi-button -icon -flat" id="button-br-column-customization" aria-label="Column Customization" data-tooltip="Column Customization" data-position="top">
+                  <button class="chi-button -portal -icon -primary -flat" id="button-p-filters" aria-label="Filters" data-tooltip="Filters" data-position="top">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-filter"></i>
+                    </div>
+                  </button>
+                </div>
+                <div class="chi-toolbar__filters-mobile">
+                  <button class="chi-button -portal -icon -primary -flat chi-drawer__trigger" id="drawer-trigger-bp1" data-target="#drawer-1" aria-label="Button action">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-filter"></i>
+                    </div>
+                  </button>
+                </div>
+              </div>
+            </div>
+            <div class="chi-toolbar__end">
+              <div class="chi-toolbar__actions">
+                <div class="chi-toolbar__actions-desktop">
+                  <button class="chi-button -portal -icon -primary -flat" id="button-p-download" aria-label="Download" data-tooltip="Download" data-position="top">
```

Comment:
-portal class is not needed anymmore, please remove

Reply:
Solved.